packages/subscription-manager/src/components/UserSettings/UserSettings.tsx

```diff
@@ -32,11 +39,20 @@ type NoticeProps = {
 };
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars -- until we start using any of these props
-const UserSettings = ( { value = {}, loading }: UserSettingsProps ) => {
+const UserSettings = ( { value = {}, loading, userId }: UserSettingsProps ) => {
 	const [ formState, setFormState ] = useState< SubscriptionUserSettings >( value );
 	const { mutate, isLoading, isSuccess, error } =
 		Reader.useSubscriptionManagerUserSettingsMutation();
 	const [ notice, setNotice ] = useState< NoticeProps | null >( null );
+	const subscriberEmailAddress = useSubscriberEmailAddress();
+
+	useEffect( () => {
```

Comment:
This redirect rule seems to be related with the Subscription Manager flow, not only to the UserSettings, in this case maybe SubscriptionManagementPage would be a better place to keep this redirect logic. What do you think?